website/mailing_list/views.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+from website.project.decorators import (
+    must_be_valid_project,
+    must_have_permission,
+    must_not_be_registration,
+)
+from website.util.permissions import ADMIN
+from website.mailing_list import utils
+
+
+###############################################################################
+# View Functions
+###############################################################################
+
+@must_be_valid_project
+@must_have_permission(ADMIN)
+@must_not_be_registration
+def get_node_mailing_list(node, auth, **kwargs):
+    return format_node_data_recursive([node], auth.user)
+
+@must_be_valid_project
+@must_have_permission(ADMIN)
+@must_not_be_registration
+def enable_mailing_list(node, **kwargs):
+    utils.celery_create_list(node._id)
+    node.mailing_enabled = True
+    node.save()
+    return {'message': 'Successfully enabled mailing lists', 'success': True}
+
+
+@must_be_valid_project
+@must_have_permission(ADMIN)
+@must_not_be_registration
+def disable_mailing_list(node, **kwargs):
+    utils.celery_delete_list(node._id)
+    node.mailing_enabled = False
```

Comment:
>the following scenario

Valid concern.

>a. Do we have a mechanism to retry deletion later if it fails?

Yes, but it requires `mailing_enabled` being set to `True`. Will update.

>b. Will users still be able to receive emails from a list that they thought was gone?

Only while the delete is pending / unsuccessful

>c. Are there any other issues that could arise here?

*Here*? A large number of admins toggling list enablement status repeatedly could maybe cause queuing issues